DESCRIPTION

```diff
@@ -32,7 +32,9 @@ LazyData: true
 Imports: 
     data.table,
     dplyr,
+    GAMBLR.helpers,
```

Comment:
:fry:

Reply:
@vladimirsouza can you please also fix this since you already have this branch checked out?

Reply:
That's right, there is no need for `GAMBLR.helpers` in `Imports` and it should be dropped from the `DESCRIPTION` file

Reply:
Solved.

Reply:
Thanks!!!